src/request.rs

```diff
@@ -15,46 +15,78 @@ pub(crate) enum Method<T: Serialize> {
 #[cfg(not(target_arch = "wasm32"))]
 pub(crate) async fn request<Input: Serialize, Output: DeserializeOwned + 'static>(
     url: &str,
-    apikey: &str,
+    apikey: &Option<String>,
```

Comment:
```suggestion
    apikey: Option<&str>,
```

It’s strange to store a reference to an option. Usually we store a reference to the data it contains.